tools/bowtie2.md

```diff
@@ -0,0 +1,138 @@
+# bowtie2 tutorial + a little bit about IVG
+written by: [trip - flavia tripon](https://github.com/triponfl)
+
+[~20 minutes] Bowtie2 aligns sequencing reads to reference sequences (so you will be able to use the output file to see coverage of a reference sequence if you add it into IVG). In relation to the VIRUSxDISCOVERY project, the aim is to determine reads from the raw SRA (SRR/ERR) file that align to the contig you've determined.
+
+**Tutorial Objective**: Align SRR11741344 reads (raw) to SRR11741344_2340 (the contig you found related to your virus)
+
+## Input / Prerequisites
+**Required packages**: sra-tools, bowtie2
```

Comment:
IGV is also used in this tutorial, add links to the tool websites at the end of the document.